### PR TITLE
- Converted Order.status to a new enum type seen in Status.java

### PR DESCRIPTION
**Pull request by GalacticWafer**
- Converted boolean Order.isSale to new enum Order.eventType seen in EventType.java
- removed redundant throws from declarations in the associated classes

### CODE REVIEW

File: src/main/java/customerrelationsmanagement/ChartType.java [hunk 1]
```diff
@@ -8,6 +8,11 @@
 	TOP_CUSTOMERS,
 	TOP_PRODUCTS;
 	
+	/**
```
Comment (OutbreakSource) [inline src/main/java/customerrelationsmanagement/ChartType.java:11]: Glad to see more comments in the Javadoc

File: src/main/java/customerrelationsmanagement/Emailer.java [hunk 2]
```diff
@@ -152,19 +153,17 @@ public void processEmails(Crud crud)
...
 		
 		for(Message currentMessage: messages) {
 			Order order = null;
 			
 			String[] messageText =
 			 getTextFromMessage(currentMessage).trim().split("\n");
 			
-			Timestamp timestamp =
-			 new Timestamp(currentMessage.getSentDate().getTime());
+			DateTime timestamp =
```
Comment (OutbreakSource) [inline src/main/java/customerrelationsmanagement/Emailer.java:165]: Hopefully, this is the actual solution to our issue with the times in SQL. Good find Malcolm!

File: src/main/java/customerrelationsmanagement/Order.java [hunk 3]
```diff
@@ -33,25 +32,25 @@
...
 	private String email;
-	private boolean isSale;
+	private EventType eventType;
 	private final String location;
 	private String messageText;
 	public String orderId;
 	private ArrayList<Product> products;
 	private static final SecureRandom rand = new SecureRandom();
-	private int status;
+	private Status status;
```
Comment (OutbreakSource) [inline src/main/java/customerrelationsmanagement/Order.java:41]: Just searched up the class on Google, good find! Didn't know there was a class that did exactly what we needed to do. Perfect replacement of our implementation of the status with just int.

**Review by OutbreakSource — APPROVED** [review]
Good work. Love the refactor/cleanup here. Good find on the stuff like the Status Java class I didn't know existed

**Review by Hamadesigns — APPROVED** [review]
Good work on refactoring code, this is the first time I'm seeing status class, overall everything looks to be in order. good work on commenting as well.

**Comment by Adam-Prieto** [comment]
Reviewed 11/19.

**Pull request merged**